portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String textValue = request.getParameter("Name");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("Name: " + textValue);
+
+     // Get the value entered in the form.
+    textValue = request.getParameter("Email");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("Email: " + textValue);
+
+     // Get the value entered in the form.
+    textValue = request.getParameter("Message");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("Meesage: " + textValue);
+
+    // Write the value to the response so the user can see it.
+    response.getWriter().println("You submitted: " + textValue);
+
+    response.sendRedirect("http://despindola-sps-spring21.appspot.com");
```

Comment:
You'd be better to change this to 
 response.sendRedirect("/");
or
 response.sendRedirect("/contact.html");

i.e. not to a different server, so that you can test it locally